Modules/HR/Http/Controllers/ResourcesController.php

```diff
@@ -17,11 +17,15 @@ public function index()
         return view('hr::guidelines-resources.index', compact('jobs'));
     }
 
-    public function show(Job $job)
+    public function show($jobId)
     {
+        $job = Job::find($jobId);
+
```

Comment:
```suggestion
    public function show(Job $job)
    {

```
This will directly give you the object Job whose Id you have passed. You just need to check if in route the variable name should also be job i.e same as the variable name you have used in Controller